_posts/2017-05-30-database-encryption.markdown

```diff
@@ -32,39 +32,60 @@ We looked at few gems, mainly [attr_encrypted](https://github.com/attr-encrypted
 
 In the end, we decided to use **Symmetric Encryption**, mainly based on their robust [docs](https://rocketjob.github.io/symmetric-encryption/), ease of use and easy integration with other libraries (in our case ActiveRecord). They provide some useful [rake tasks](https://rocketjob.github.io/symmetric-encryption/rake_tasks.html) and also describe how to configure this gem for apps inside/outside Heroku. Overall `symmetric_encryption` seemed really robust, reliable and 🔒 .
 
-## Using `symmetric_encryption`
-Symmetric encryption provides a seemless integration with `ActiveRecord`, they provide `attr_encrypted` helper method which can be used to define _encrypted_ fields so you can add following to your `ActiveRecord` model:
+Symmetric Encryption uses OpenSSL to encrypt and decrypt the data which means we are able to use any of the encryption algorithms supported by OpenSSL.
+
+## Configuration
+As mentioned above, we can configure our encryption algorithm for being deployed in Heroku and outside of it. The main difference is, outside of Heroku were we may have access to file system outside of source code we can configure encryption differently.
+
+### Outside of Heroku
+`symmetric_encryption` expects the encryption key to be stored on a file on disk outside of source code and expects OS to deal with security of that file.
+
+### In Heroku
+Since we don't have access to file system in Heroku outside of source code, `symmetric_encryption` suggests to add encryption key in environment variable BUT in order to do that we need to encrypt the encryption key. Secret used for encrypting encryption key can be committed into source code this means for someone in order to be able to decode the data:
+- They have to be able to access our database
+- They need to be able to access our source code
+- They need to be able to access Heroku's configuration
```

Comment:
I described that a little [here](https://github.com/artsy/artsy.github.io/pull/352/files#diff-a742c383aaadc35ca085c1e465eb07dfR12) but do you think I need to make it more obvious?